src/test/scala/uk/gov/nationalarchives/consignmentexport/MainSpec.scala

```diff
@@ -115,6 +115,36 @@ class MainSpec extends ExternalServiceSpec {
     source.close()
   }
 
+  "the export job" should "create directories for folders" in {
```

Comment:
Maybe reword test tile slightly to: ... create directories for empty 'folder' type ...